### DISCUSSION TITLE
Looking for guidance on contributions

### DISCUSSION THREAD

**Discussion by TrevinTeacutter** [vectordotdev/vector#8708] [General]
Hello, I'm just looking for a good place to have a conversation around whether it makes sense for us to try to contribute? I wanted to post a discussion before I dived into your discord.

For context, I'm from a team in an MSSP that helps customers get various sources of data generally into [Google Cloud Chronicle](https://chronicle.security/) and we were actually looking at completely redoing our internal Golang library to something that resembles Vector with some minor differences (we just now discovered Vector so good to know it seems we were on the right track). We haven't started that work beyond a PoC of sorts and we are wanting to evaluate options and just looking to get whether some of the features we would like align with your aspirations for Vector or not as that would help us make a decision.

Side note: Man are titles hard

**Comment by jszwedko** [comment]
Hey @TrevinTeacutter !

We'd love to discuss what features you are thinking about and see if they align with Vector. Would you want to just outline them here? Or setup a call? If the latter, you can email jesse.szwedko@datadoghq.com .

**Reply by TrevinTeacutter** [reply]
Probably makes more sense to have a call, I find it helps to give context around what problems you are trying to solve with feature requests. Sent you an email, so consider this discussion answered!
